logic-apps/pull-reference-data.json

```diff
@@ -619,6 +690,101 @@
         },
         "type": "If"
       },
+      "IfActiveAssetUnivariateAnomalyDetectionScenarioExists": {
+        "actions": {
+          "GetAssetUnivariateAnomalyDetectionParams": {
+            "inputs": {
+              "authentication": "@parameters('DynamicsIdentityAuthentication')",
+              "method": "GET",
+              "uri": "@{parameters('EnvironmentUrl')}data/AnomalyDetectionParams"
```

Comment:
Reminder to update entity name 
![image](https://user-images.githubusercontent.com/30123023/232752352-613e9863-261f-495d-9b50-7ece67e7e501.png)

Reply:
@denisramiros Could you please check th public name of that entity? Or is it the same?

Reply:
@denisramiros There is a typo "CounterMaxAnomalyRatioValue"